lib/telemetry_metrics.ex

```diff
@@ -319,12 +319,16 @@ defmodule Telemetry.Metrics do
   Distribution metric builds a histogram of selected measurement's values. Because of that, it is
   required that you specify the histograms buckets via `:buckets` option.
 
-  For example, given `buckets: [0, 100, 200]`, the distribution metric produces four values:
+  TThe buckets is either a list of integers, such as `[100, 200, 300]`, or a two-element tuple,
+  containing the range as first element and the step as second, such as `{100..300, 100}`, which
+  emits the same buckets as `[100, 200, 300]`.
+
+  Given `buckets: [100, 200, 300]`, the distribution metric produces four values:
 
     * number of measurements less than or equal to 0
```

Comment:
Is this still true? It would be strange to `[100]` also count measurements less than `0`.